### PR TITLE
Add Prohashing pool

### PR DESCRIPTION
**Pull request by vtheskeleton**
lets begin with this:

image

Github defaulted to sus sus amongus

next up: I added the Prohashing pool. This shouldn't be merged before we decide that the public beta was a success

### CODE REVIEW

File: src/setup.js [hunk 1]
```diff
@@ -99,27 +101,51 @@ In order for this to work, you'll need to have the Discord desktop app installed
...
+			spinner.fail()
+			console.log(chalk.bold.red("Could not find your Rig ID! Please make sure that you have mined for at least 5 minutes using Salad's official application after restarting it."));
 			setTimeout(() => {
 				continueSetup()
 			}, 3500);
 			return;
 		}
+		if (!id) {
+			spinner.fail()
+			console.log(chalk.bold.red("Could not find your id for Prohashing! You can use the Automatic (Auth Token) method or manual instead."));
```
Comment (TheBlueBurger) [inline src/setup.js:126]: Maybe tell the user it's possible to press override in salad and start mining a few minutes? That way would work right? Unable to test right now

File: src/setup.js [hunk 2]
```diff
@@ -99,27 +101,51 @@ In order for this to work, you'll need to have the Discord desktop app installed
...
+			let skipProhashing = await inquirer.prompt([{
+				type: 'confirm',
+				name: 'skipProhashing',
+				message: `Continue without the id for Prohashing? ${chalk.yellow.bold("If you continue without the id for Prohashing, you cannot use the Prohashing pool which has several advantages.")}`,
+				default: false
+			}]).then(function(answers) {
+				if (!answers.skipProhashing) {
+					setTimeout(() => {
+						continueSetup()
+					}, 3500);
```
Comment (TheBlueBurger) [inline src/setup.js:136]: Why do you need to wait if the user has nothing to read while waiting?

**Review by TheBlueBurger — CHANGES_REQUESTED** [review]

File: src/setup.js (hunk 1 above)
Comment (vtheskeleton) [reply src/setup.js:126]: We cant confirm it will force it onto the new pool

File: src/setup.js (hunk 2 above)
Comment (vtheskeleton) [reply src/setup.js:136]: because i copied your code

**Comment by TheBlueBurger** [comment]
There seems to just be 2 issues until this can be merged, I can look into them if my pc fan decides to start working

**Comment by vtheskeleton** [comment]
Which Issues?

**Comment by vtheskeleton** [comment]
i forgor 💀

**Pull request merged**